modules/guides/pages/connect.adoc

```diff
@@ -0,0 +1,422 @@
+= Connect to Couchbase
+:description: How to connect to a Couchbase Cluster.
+:page-topic-type: guide
+:page-toclevels: 2
+:tabs:
+:github: Click the GitHub button icon:github[] to view this code in context.
+
+include::partial$example-attributes.adoc[]
+
+[abstract]
+{description}
+
+== Introduction
+
+Connecting to Couchbase can be done in several ways.
+This guide will take you through some of the most common methods used to access a Couchbase cluster with an SDK client or cli tool.
+
+A Couchbase xref:learn:clusters-and-availability/clusters-and-availability.adoc[Cluster] is a combination of multiple server nodes, which can be accessed by users or applications with a xref:learn:security/usernames-and-passwords.adoc[username and password].
+Each server node can also be its own cluster or join an existing multi-node setup.
+
+Couchbase uses xref:learn:security/roles.adoc[Role Based Access Control (RBAC)] to control access to its various xref:learn:services-and-indexes/services/services.adoc[services].
+A user or application can connect to a cluster and use these services, assuming that valid credentials with relevant access roles are provided.
+
+include::partial$before-you-begin.adoc[]
+
+include::partial$clients.adoc[tags=!refs-ui]
+
+== Connecting to Couchbase Server
+
+Couchbase Server can be configured to run with _unencrypted_ or _encrypted_ network access. 
+When running Couchbase in a production environment, the latter is always recommended.
+
+=== Basic Auth
+
+To connect to a standalone or Docker installation with _unencrypted_ network access, setup a user with appropriate access levels and a secure password.
```

Comment:
Repeat inline links to explanatory topic(s) for RBAC ("appropriate access links") and password. (I know they're linked above, but just to avoid confusion at this point.)